scripts/pre_push_hook.py

```diff
@@ -326,15 +345,18 @@ def get_refs():
     return ref_list
 
 
-def start_linter(files):
+def start_linter(files: List[bytes]) -> int:
     """Starts the lint checks and returns the returncode of the task."""
-    task = subprocess.Popen(
-        [PYTHON_CMD, '-m', LINTER_MODULE, LINTER_FILE_FLAG] + files)
+    cmd_list: List[Union[str, bytes]] = [
```

Comment:
Let's get rid of these "union" things and have single-typed cmd lists throughout all the scripts/ files. I don't see the need to maintain a union for these.

I feel like, in general, str are more standard, but please let me know if I'm wrong here and doing this causes some issues. You might want to read up a bit on what Python 3 prefers, too.

Reply:
I did some changes PTAL! ( tests and pre_commit_hook are working fine with new changes )